.travis.yml

```diff
@@ -71,24 +69,13 @@ matrix:
 
     - os: osx
       osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=osx-10-13-cxx14
+      env: PROJECT_DIR=examples/acf TOOLCHAIN=osx-10-13-cxx14 VERBOSE=0
 
-    # WebP i386 build is broken
+    # Upload from local machine
+    # FIXME: modules/core/src/parallel_impl.cpp:396:21: error: implicitly declaring library function '_mm_pause' with type 'void ()'
     # - os: osx
     #   osx_image: xcode9.2
-    #   env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3
-
-    # }
-
-    # Extra {
-
-    - os: osx
-      osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7
```

Comment:
This one is working, no need to remove.

Reply:
Okay.  I removed it since it wasn't in pkg.template or pkg.opencv.  I can add it back.

Reply:
I've added those in an `# Extra { ` `# }` block